src/index.ts

```diff
@@ -251,20 +280,31 @@ const PersistedQueriesPlugin: PostGraphilePlugin = {
   },
 
   // For regular HTTP requests
-  "postgraphile:httpParamsList"(paramsList, { options }) {
+  "postgraphile:httpParamsList"(paramsList, { options, req }) {
     return paramsList.map((params: any) => {
+      const allowUnpersistedOperations =
+        options.allowUnpersistedOperation || (() => false);
```

Comment:
This puts unnecessary load on the garbage collector by creating a lambda function that'll be thrown away moments later. Better to solve it below: